aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,302 @@
+    CloudFrontClient = Any
+
+
+class Environment(Enum):
+    """Deployment environments"""
+    STAGING = "staging"
+    PRODUCTION = "production"
+
+
+class CloudFrontInvalidationError(Exception):
+    """CloudFront invalidation operation error"""
+    pass
```

Ruff (0.11.9)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)